server/bin/import_test_data.rb

```diff
@@ -37,12 +37,13 @@
 
 cp = Candlepin.new('admin', 'admin', nil, nil, 'localhost', 8443)
 
+print "\nCreating owners\n"
 def create_owner(cp, new_owner)
   owner_name =  new_owner['name']
   displayName = new_owner['displayName']
 
-  puts "owner: #{owner_name}"
-  puts "\t displayName: #{displayName}"
+  print "owner: #{owner_name}\n"
+  print "\t displayName: #{displayName}\n"
```

Comment:
puts isn't atomic and triggers two print operations (message and new line), where print is (at least, as far as the printing bits go). This is why while importing, we'll occasionally see two threads print on the same line with a new line between them, rather than printing on their own lines.

I had fixed this once a couple years back, but it got lost/reverted somewhere along the way.